core/print.go

```diff
@@ -23,6 +23,9 @@ import (
 	"strings"
 )
 
+// Number of spaces to use when indenting.
+var IndentSize int
```

Comment:
Also make the default value here to be 4, for other things that use 'core' without going through the buildifier binary